papas/reconstruction/src/PFReconstructor.cpp

```diff
@@ -12,9 +12,9 @@
 #include "papas/datatypes/Path.h"
 #include "papas/datatypes/Track.h"
 #include "papas/graphtools/Edge.h"
+#include "papas/reconstruction/BuildPFBlocks.h"
```

Comment:
They aren't new (just replacements of old) and you are right they are now redundant. I have a header "review" down as an issue to look at soon. Have removed this